admin/tabs/preferences/JsonBlobModel.js

```diff
@@ -0,0 +1,125 @@
+
+    persistWith = {localStorageKey: 'xhAdminJsonBlobState'};
+
+    @managed
+    gridModel = new RestGridModel({
+        persistWith: this.persistWith,
+        colChooserModel: true,
+        enableExport: true,
+        selModel: 'multiple',
+        store: {
+            url: 'rest/jsonBlobAdmin',
+            reloadLookupsOnLoad: true,
+            fields: [
+                {
+                    name: 'owner'
+                },
+                {
+                    name: 'acl',
+                    displayName: 'ACL'
+                },
+                {
+                    name: 'name',
+                    required: true
+                },
+                {
+                    name: 'type',
+                    lookupName: 'types',
+                    required: true,
+                    enableCreate: true
+                },
+                {
+                    name: 'value',
+                    type: 'json'
+                },
+                {
+                    name: 'description'
+                },
+                {
+                    name: 'dateCreated',
+                    type: 'date',
+                    editable: false
+                },
+                {
+                    name: 'lastUpdated',
+                    type: 'date',
+                    editable: false
+                },
+                {
+                    name: 'lastUpdatedBy',
+                    editable: false
+                }
+            ]
+        },
+        toolbarActions: [
+            addAction,
+            editAction,
+            cloneAction,
+            deleteAction
+        ],
+        menuActions: [
+            addAction,
+            editAction,
+            cloneAction,
+            deleteAction
+        ],
+        prepareCloneFn: ({clone}) => clone.name = `${clone.name}_CLONE`,
+        sortBy: ['owner', 'name'],
+        groupBy: 'type',
+        unit: 'blob',
+        filterFields: ['name', 'owner', 'type', 'value', 'description'],
+        columns: [
+            {field: 'owner', width: 200},
+            {field: 'acl', width: 80},
+            {field: 'name', width: 200},
+            {field: 'type', width: 200},
+            {field: 'description', width: 200},
+            {field: 'value', flex: 1, renderer: v => truncate(v, {length: 500})},
+            {field: 'dateCreated', ...dateTimeCol, hidden: true},
+            {field: 'lastUpdated', ...dateTimeCol, hidden: true},
+            {field: 'lastUpdatedBy', width: 160, hidden: true}
+        ],
+        editors: [
+            {field: 'owner'},
+            {field: 'acl'},
+            {field: 'name'},
+            {field: 'type'},
+            {field: 'description', formField: {item: textArea()}},
+            {field: 'value'},
+            {field: 'dateCreated'},
+            {field: 'lastUpdated'},
+            {field: 'lastUpdatedBy'}
+        ]
+    });
+
+    @managed
+    differModel = new DifferModel(this.gridModel, 'jsonBlob');
```

Comment:
It appears the DifferModel will match local and remote records by just comparing the name, which works fine for configs and prefs, but for json blobs I think we also need to take the owner and type into account to match the records. Perhaps we should pass the matcher fn into the DifferModel constructor?